pkg/coderefs/coderefs.go

```diff
@@ -48,13 +47,18 @@ type fileSearchResults struct {
 type branch struct {
 	Name             string
 	Head             string
-	UpdateSequenceId *int64
+	UpdateSequenceId *int
 	SyncTime         int64
 	SearchResults    searchResultLines
 }
 
-func Scan() {
-	dir := o.Dir.Value()
+func Scan(opts options.Options) {
```

Comment:
passing options gets rid of global state and sets us up to make `ld-find-code-refs` usable as a go package in the future.